buildpipeline/DotNet-Trusted-Publish-Symbols.json

```diff
@@ -177,8 +215,8 @@
       "value": "false",
       "allowOverride": true
     },
-    "PB_BuildType": {
-      "value": "Release",
+    "ConfigurationGroup": {
+      "value": "$(PB_BuildType)",
```

Comment:
I think it would be good to give `PB_BuildType` a default, so we don't end up with `$(PB_BuildType)` showing up literally at some point.